packages/adapter-drizzle/src/lib/mysql.ts

```diff
@@ -100,15 +88,17 @@ export function MySqlDrizzleAdapter(
     schema
 
   return {
-    async createUser(data: Omit<AdapterUser, "id">) {
-      const id = randomUUID()
+    async createUser(data: AdapterUser) {
+      const hasDefaultId = getTableColumns(usersTable)["id"]["hasDefault"]
```

Comment:
is the `getTableColumns` needed? Do Drizzle recommend not using the `_` attribute on the table directly?

Reply:
If we use `_` then we get `undefined`. That's why I chose to use `getTableColumns`